src/main/java/com/github/kiulian/downloader/model/search/SearchResultPlaylistDetails.java

```diff
@@ -0,0 +1,45 @@
+package com.github.kiulian.downloader.model.search;
+
+import java.util.LinkedList;
+
+import com.alibaba.fastjson.JSONArray;
+import com.alibaba.fastjson.JSONObject;
+
+public class SearchResultPlaylistDetails extends AbstractSearchResultList {
+
+    private String playlistId;
+    private int videoCount;
+
+    public SearchResultPlaylistDetails() {}
```

Comment:
why do we need empty constructor here and in others classes `...Details` ?

Reply:
and lets keep `final` fields where possible

Reply:
> why do we need empty constructor here and in others classes ...Details ?

We do not !
> and lets keep final fields where possible

Agree